Tests/DangerTests/BitBucketCloudTests.swift

```diff
@@ -70,6 +70,18 @@ final class BitBucketCloudTests: XCTestCase {
         }
     }
 
+    func testCommentUserWithoutAccountId() {
+        do {
+            let comment = try JSONDecoder().decode(BitBucketCloud.Comment.self, from: Data(BitBucketCloudCommentUserWithoutAccountIdAndNickname.utf8))
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/BitBucketCloudTests.swift;
  Line: 75;
-->

- :warning: Line should be 140 characters or less: currently 150 characters (`line_length`)